src/Statuses/Status.ts

```diff
@@ -22,15 +22,15 @@ export class Status {
      * @type {Status}
      * @memberof Status
      */
-    public static DONE: Status = Status.makeDone();
+    public static DONE: Status = new Status(new StatusConfiguration('x', 'Done', ' ', true, StatusType.DONE));
 
     /**
      * A default status of empty, used when things go wrong.
      *
      * @static
      * @memberof Status
      */
-    public static EMPTY: Status = Status.makeEmpty();
+    public static EMPTY: Status = new Status(new StatusConfiguration('', 'EMPTY', '', true, StatusType.EMPTY));
```

Comment:
I agree with the SonarCloud that static properties should be readonly. It was an error previously that they weren't.

Reply:
The capitalised name `EMPTY` was intended to convey that it is a constant.

Reply:
Also, I think there description of this commit may suggest a misunderstanding here:

> refactor: . inline DONE, EMPTY, TODO static methods

They weren't methods, they were properties/fields, so only initialised once...

Reply:
> I agree with the SonarCloud that static properties should be readonly. It was an error previously that they weren't.

Right, fixed.

Reply:
> The capitalised name `EMPTY` was intended to convey that it is a constant.

Sorry, I don't get this. Is there something to do here?

Reply:
> Sorry, I don't get this. Is there something to do here?

No, adding the readonly is fine.